keyboards/ergodox_ez/readme.md

```diff
@@ -31,3 +31,59 @@ To flash with ´teensy-loader-cli´:
 
   - Press the Reset button by inserting a paperclip gently into the reset hole
     in the top right corder.
+
+## Settings
+
+You may want to enable QMK_KEYS_PER_SCAN because the Ergodox has a relatively
+slow scan rate. It's been tested with 8 and 16, both work.
+
+## Steno support
+
+Steno support is an experimental-ish feature, mostly useful with
+Plover:
+	https://github.com/openstenoproject/plover/releases
+
+Steno support works by using a virtual serial port and pretending
+to be a hardware steno machine. While steno support uses very large
+chords, the virtual serial form does *not* require NKRO support.
+NKRO is needed only to support sending lots of keys at once over
+USB HID interfaces.
```

Comment:
This kind of makes it sound like USB HID NKRO would not be useful for Steno... but per user reports, it is. Like, the new Steno support is cool, but it is _one_ way to use Plover -- NKRO would still work, would it not?

If so, could you clarify this point?